python/example_robot_data/robots_loader.py

```diff
@@ -50,19 +50,29 @@ class RobotLoader(object):
     path = ''
     urdf_filename = ''
     srdf_filename = ''
+    sdf_filename = ''
     urdf_subpath = 'robots'
     srdf_subpath = 'srdf'
+    sdf_subpath = ''
     ref_posture = 'half_sitting'
     has_rotor_parameters = False
     free_flyer = False
     verbose = False
 
     def __init__(self):
-        urdf_path = join(self.path, self.urdf_subpath, self.urdf_filename)
-        self.model_path = getModelPath(urdf_path, self.verbose)
-        self.urdf_path = join(self.model_path, urdf_path)
-        self.robot = RobotWrapper.BuildFromURDF(self.urdf_path, [join(self.model_path, '../..')],
-                                                pin.JointModelFreeFlyer() if self.free_flyer else None)
+        if self.urdf_filename:
+            df_path = join(self.path, self.urdf_subpath, self.urdf_filename)
+            builder = RobotWrapper.BuildFromURDF
```

Comment:
I would add a warning message if the developer has defined `sdf_filename`. I would include the following code:
```python
  if self.sdf_filename:
    print("Warning: we are building the robot model from the URDF")
    print("If you wish to build the model from the SDF", then do not define the 'urdf_filename'")
```